src/content/docs/tutorial-app-performance/fixing-high-latency.mdx

```diff
@@ -5,82 +5,92 @@ tags:
+Once you understand what causes latency, you can use the **Summary** and **AJAX** pages in <InlinePopover type="browser" /> to start your root cause analysis. In general, we make the following recommendations to optimize your site:
+
+* Use a content delivery network (CDN). CDNs can adjust where content is being served from, or distribute them across differe servers.
+* Optimize above-the-fold content.
+* Evaluate your web hosting service.
+* Minimize JavaScript execution time.
+* Use a caching plugin.
+
+## Diagnose high latency [#latency-example]
 
-## An example of diagnosing high latency [#latency-example]
-Once you're familiar with the common causes of latency, you can narrow down the specific issue that is impacting your site. Let's take a look at a service that is having issues with latency.
+Let's take a look at a service that is having issues with latency.
 
 <img
   alt="A screenshot of the summary page with a poor LCP score"
   title="Abnormal LCP score"
   src={journeyAbnormalLCP}
 />
 
-Here, you see the **Summary** page for a service showing a **Largest Contentful Paint** of 8.44 seconds. Since everything else falls under acceptable parameters, you can narrow down the cause of your latency as slow page load. You could select **Session trace** page for a specific view of what might be causing the issues with your LCP. This would be a good place to start a search to find what is causing your latency issues.
-
-You could also select the **Largest Contentful Paint** section itself to go to the **Web vitals** page for a more generalized overview of errors based on page URLs, device type, browser type, and more. Once you find the source of the issue using this method, you can repair the problem and get your user experience back to the levels that keep your customers happy.
-
-
-<SideBySide>
-    <Side>
-        <DocTiles variant="labs">
-            <DocTile
-                path="/docs/journey-performance/guide-to-monitoring-core-web-vitals/"
-                label={{text: '⬅️ PREVIOUS', color: '#AFE2E3'}}    
-            >
-                Learn more about core web vitals
-            </DocTile>
-        </DocTiles>
-    </Side>
-    <Side>
-        <DocTiles variant="labs">
-            <DocTile
-                path="/docs/journey-performance/reduce-javascript-errors/"
-                label={{text: 'NEXT ➡️', color: '#AFE2E3'}}    
-            >
-                See how to reduce JavaScript errors
-            </DocTile>
-        </DocTiles>
-    </Side>
-</SideBySide>
\ No newline at end of file
+From the browser **Summary** page, you notice a service showing a **Largest Contentful Paint** of 8.44 seconds. Since everything else falls under acceptable parameters, you can narrow down the cause of your latency as slow page load. 
```

Comment:
Maybe it would be helpful to have a path and link here? Just so the user can follow along a little easier if they want to.